.github/workflows/seo.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: SEO CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.8.0
+        with:
+          fail: true
+          args: --max-redirects 10 -a 403,500,503 .
+  seo_spy_orphan_pages:
+    runs-on: ubuntu-latest
+    name: Check orphan pages
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Checkout SEO Spy
+        uses: actions/checkout@v3
+        with:
+          repository: 3mdeb/seo-spy
+          path: seo-spy
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r seo-spy/requirements.txt
+          pip install -r requirements.txt
+      - name: Serve site
+        run: |
+          mkdocs serve &
+          sleep 15
```

Comment:
@artur-rs The `&` makes this command asynchronous, so we have to wait for the server to build. I took time average time of build and I added 10 seconds. Currently, there is no risk of a race condition, but when we will reach thousands of markdown files here, it might be the issue.

Reply:
How about `mkdocs build` and then staring server in `site` via `python3 -m http.server` ?

Reply:
Sitemap needs to be `sed`ed but it works well. https://github.com/Dasharo/docs/pull/614/commits/5b59db761c40b20be0f98ce2e0ebfba7e061e99c